arm/Microsoft.ApiManagement/service/products/readme.md

```diff
@@ -21,12 +21,12 @@ The following resources are required to be able to deploy this resource.
 | Parameter Name | Type | Default Value | Possible Values | Description |
 | :-- | :-- | :-- | :-- | :-- |
 | `apiManagementServiceName` | string |  |  | Required. The name of the of the Api Management service. |
-| `approvalRequired` | bool |  |  | Optional. Whether subscription approval is required. If false, new subscriptions will be approved automatically enabling developers to call the product�s APIs immediately after subscribing. If true, administrators must manually approve the subscription before the developer can any of the product�s APIs. Can be present only if subscriptionRequired property is present and has a value of false. |
+| `approvalRequired` | bool |  |  | Optional. Whether subscription approval is required. If false, new subscriptions will be approved automatically enabling developers to call the productÆs APIs immediately after subscribing. If true, administrators must manually approve the subscription before the developer can any of the productÆs APIs. Can be present only if subscriptionRequired property is present and has a value of false. |
```

Comment:
Something is happening with special characters here

Reply:
done